### PR TITLE
add content

### PR DESCRIPTION
**Pull request by lourieder**

### CODE REVIEW

File: doc/source/rst.rst [hunk 1]
```diff
@@ -2,14 +2,174 @@
...
+
+    .. code-block:: 
+
+        .. figure:: /_static/images/filename
+
+            :alt: This is your title 
+
+
+
+Integrating Admonitions
```
Comment (vladimirhasko) [inline doc/source/rst.rst:31]: show all admonition types as examples how they really look like.

File: doc/source/rst.rst [hunk 2]
```diff
@@ -2,14 +2,174 @@
 Understanding the RST
```
Comment (vladimirhasko) [inline doc/source/rst.rst:2]: please add also info about how to work with intersection links within same page or intesection links from other page but in same document.
How to use anchors , etc
Also how to work with ordered and unordered list.

**Review by vladimirhasko — CHANGES_REQUESTED** [review]

File: doc/source/rst.rst [hunk 3]
```diff
@@ -2,14 +2,174 @@
...
+
+   .. note:: This is an easy way to implement pictures espacially in tables. Ideally the file path would be at the bottom of your file. 
+
+The more elegant way is to use figures with following syntax:
+
+    .. code-block:: 
+
+        .. figure:: /_static/images/filename
+
+            :alt: This is your title
```
Comment (gtema) [inline doc/source/rst.rst:27]: "alt" is so to say a hidden title (visible on hover if at all). The most proper way is to simply include title in the text form inside of the macros body (https://sphinxguide.readthedocs.io/en/latest/fig-num/fig.html)

**Comment by vladimirhasko** [comment]
Recheck

**Comment by kucerakk** [comment]
merged with https://github.com/opentelekomcloud-docs/docs_on_docs/pull/22

**Pull request closed without merging**